Shell/Options.cpp

```diff
@@ -1541,6 +1541,16 @@ void Options::init()
     _demodulationRedundancyCheck.onlyUsefulWith(Or(_forwardDemodulation.is(notEqual(Demodulation::OFF)),_backwardDemodulation.is(notEqual(Demodulation::OFF))));
     _demodulationRedundancyCheck.addProblemConstraint(hasEquality());
 
+    _demodulationPrecompiledComparison = BoolOptionValue("demodulation_precompiled_comparison","dpc",false);
+    _demodulationPrecompiledComparison.description=
+       "Precompiles ordering constraints on unorientable demodulators which results in less overhead when actually comparing."
+       "Only works with KBO currently.";
```

Comment:
According to the comments of the PR, isn't this outdated? (I.e., works also for LPO?)

Reply:
Yes, thanks for catching it!